src/Library/Character.cs

```diff
@@ -60,9 +60,29 @@ public void AddSpell(Spell spell)
                 Console.WriteLine("You're not a wizard Harry");
             }
         }
-                    
+
+        public int UseSpell(Spell spell)
+        {
+            if(spell in this.bookOfSpells)
```

Comment:
Creo que el _in_ no existe en C#. Yo hubiese puesto:
if(this.bookOfSpells.contains(spell))